### PR TITLE
Update JCAPI for Workday and misc other changes

### PR DESCRIPTION
**Pull request by run-o**
Main changes:
- lots of docs changes
- workday endpoints
- enable following missing traversal endpoints:
  api/v2/systems/{systemID}/commands
  api/v2/systemgroups/{systemgroup_ID}/commands
  api/v2/users/{user_ID}/systemgroups
  api/v2/systems/{system_ID}/usergroups
- misc cleanups

### CODE REVIEW

File: v2/policytemplates_api.go [hunk 1]
```diff
@@ -28,8 +28,8 @@ type PolicytemplatesApiService service
 
 
 /* PolicytemplatesApiService Get a specific Policy Template
- This endpoint returns a specific policy template.
- * @param ctx context.Context Authentication Context 
+ This endpoint returns a specific policy template.  #### Sample Request &#x60;&#x60;&#x60;  curl -X GET https://console.jumpcloud.com/api/v2/policies/{Policy_ID}\\   -H &#39;Accept: application/json&#39; \\   -H &#39;Content-Type: application/json&#39; \\   -H &#39;x-api-key: {API_KEY}&#39; &#x60;&#x60;&#x60;
```
Comment (jpvajda) [inline v2/policytemplates_api.go]: @run-o  This line looks a little wonky....

File: v2/policytemplates_api.go [hunk 2]
```diff
@@ -90,34 +90,35 @@ func (a *PolicytemplatesApiService) PolicytemplatesGet(ctx context.Context, id s
...
 	}
 
 
 	return successPayload, localVarHttpResponse, err
 }
 
 /* PolicytemplatesApiService Lists all of the Policy Templates
- This endpoint returns all policy templates.
- * @param ctx context.Context Authentication Context 
+ This endpoint returns all policy templates.  #### Sample Request &#x60;&#x60;&#x60; curl -X GET https://console.jumpcloud.com/api/v2/policytemplates \\   -H &#39;Accept: application/json&#39; \\   -H &#39;Content-Type: application/json&#39; \\   -H &#39;x-api-key: {API_KEY}&#39;   &#x60;&#x60;&#x60;
```
Comment (jpvajda) [inline v2/policytemplates_api.go]: @run-o another line that has some weird characters in it.

File: v2/samba_domains_api.go [hunk 3]
```diff
@@ -28,9 +28,9 @@ type SambaDomainsApiService service
 
 
 /* SambaDomainsApiService Delete Samba Domain
- This endpoint allows you to delete a samba domain from an LDAP server.
- * @param ctx context.Context Authentication Context 
- @param ldapserverId Unique identifier o f the LDAP server.
+ This endpoint allows you to delete a samba domain from an LDAP server.  ##### Sample Request &#x60;&#x60;&#x60; curl -X DELETE https://console.jumpcloud.com/api/v2/ldapservers/{LDAP_ID}/sambadomains/{SAMBA_ID} \\   -H &#39;Accept: application/json&#39; \\   -H &#39;Content-Type: application/json&#39; \\   -H &#39;x-api-key: {API_KEY}&#39; &#x60;&#x60;&#x60;
```
Comment (jpvajda) [inline v2/samba_domains_api.go]: another one @run-o

File: v2/samba_domains_api.go [hunk 4]
```diff
@@ -103,27 +103,28 @@ func (a *SambaDomainsApiService) LdapserversSambaDomainsDelete(ctx context.Conte
...
 
 
 	return successPayload, localVarHttpResponse, err
 }
 
 /* SambaDomainsApiService Get Samba Domain
- This endpoint returns a specific samba domain for an LDAP server.
- * @param ctx context.Context Authentication Context 
- @param ldapserverId Unique identifier o f the LDAP server.
+ This endpoint returns a specific samba domain for an LDAP server.  ##### Sample Request &#x60;&#x60;&#x60; curl -X GET \\   https://console.jumpcloud.com/api/v2/ldapservers/ldapservers/{LDAP_ID}/sambadomains/{SAMBA_ID} \\   -H &#39;Accept: application/json&#39; \\   -H &#39;Content-Type: application/json&#39; \\   -H &#39;x-api-key: {API_KEY}&#39;   &#x60;&#x60;&#x60;
```
Comment (jpvajda) [inline v2/samba_domains_api.go]: another @run-o

**Comment by run-o** [comment]
@jpvajda Can you change your review to approved please?
Also I've reverted your last commit (manual update of the sample request in one of the docs).

**Pull request merged**